lib/remote-api-caller-helper.ts

```diff
@@ -0,0 +1,59 @@
+import { CryptoUtil } from "./crypto-util.js";
+import { RemoteAPICaller } from "./gmd-api-caller.js";
+
+
+
+
+export class RemoteAPICallerHelper extends RemoteAPICaller {
+    //Latest block
+    async getBlockNumber(): Promise<number> {
+        const data = await this.apiCall('get', { requestType: 'getBlock' } as Record<string, string>)
+        return data.height as number;
+    }
+
+    async getBalance(rsAccount: string): Promise<string> {
+        const data = await this.apiCall('get', { requestType: 'getBalance', account: rsAccount } as Record<string, string>);
+        const nqt = (data as unknown as IGetBalanceResponse).balanceNQT;
+        return CryptoUtil.Crypto.NqtToGmd(nqt);
+    }
+
+    async getTransactions(outbound: boolean, rsAccount: string, pageSize = 10, page = 0, type: number | null = null, subtype: number | null = null) {
+        const request: IGetTransactionRequest = {
+            requestType: 'getTransactionsBulk',
+            pageSize: pageSize,
+            page: page
+
+        };
+        if (type !== null) {
+            request.filterByType = type;
+            if (subtype !== null) {
+                request.filterBySubtype = subtype;
+            }
+        }
+        if (outbound) {
+            request.filterBySender = rsAccount;
+        } else {
+            request.filterByReceiver = rsAccount;
+        }
+        const data = await this.apiCall('get', request as unknown as Record<string, string>);
+        return (data as unknown as IGetTransactionsResponse).Transactions;
+    }
+}
+
+export interface IGetTransactionRequest {
+    requestType: string,

```

Comment:
You can create a 
``` export type RequestType = "get" | "post" | "put"` etc and use it around.

Reply:
GMD node APIs only allows get and post.
Good suggestion, thanks.